modules/launcher/src/test/java/org/mule/module/launcher/builder/AbstractArtifactFileBuilder.java

```diff
@@ -46,12 +46,23 @@
      * Creates a new builder
      *
      * @param id artifact identifier. Non empty.
+     * @param upperCaseInExtension whether the extension is in uppercase
      */
-    public AbstractArtifactFileBuilder(String id)
+    public AbstractArtifactFileBuilder(String id, boolean upperCaseInExtension)
     {
         checkArgument(!StringUtils.isEmpty(id), "ID cannot be empty");
         this.id = id;
-        this.fileName = id + ".zip";
+        this.fileName = id + (upperCaseInExtension ? ".ZIP" : ".zip");
```

Comment:
use DefaultArchiveDeployer.ZIP_FILE_SUFFIX, and toUpperCase for the other scenario